source/Conversation.h

```diff
@@ -75,11 +86,15 @@ class Conversation {
 	int Choices(int node) const;
 	bool IsBranch(int node) const;
 	bool IsAction(int node) const;
-	const ConditionSet &Branch(int node) const;
+	const ConditionSet &Conditions(int node) const;
 	const GameAction &GetAction(int node) const;
 	const std::string &Text(int node, int choice = 0) const;
 	const Sprite *Scene(int node) const;
 	int NextNode(int node, int choice = 0) const;
+	bool ShouldSkipText(const std::map<std::string, int64_t> &vars, int node, int choice = 0) const;
```

Comment:
As with below, this is only used in ConversationPanel right now, so might it be better to define it there?

Reply:
I disagree; the conversation is in charge of the node chain. The panel renders the node chain and handles player input.

Moving this to the panel means the panel has to know more about what makes a node belong in the displayed chain, rather than trusting the conversation to do that work.

Reply:
I subscribe to the same reasoning as @tehhowch, but I could be convinced.